borsh-derive/src/internals/deserialize/struct/mod.rs

```diff
@@ -2,8 +2,8 @@ use proc_macro2::TokenStream as TokenStream2;
 use quote::quote;
 use syn::{ExprPath, Fields, Ident, ItemStruct, Path, WhereClause};
 
-use crate::{
-    attribute_helpers::{contains_initialize_with, contains_skip, field, BoundType},
+use crate::internals::{
+    attributes::{contains_initialize_with, contains_skip, field, BoundType},
     generics::{compute_predicates, without_defaults, FindTyParams},
 };
 
```

Comment:
move `fn field_deserialization_output` up one level to `borsh-derive/src/internals/deserialize/mod.rs` , as it's common between `structs` and `enums` submodules